README.md

```diff
@@ -1 +1,72 @@
-# Website
+# Pollution Detection System
+
+PDS is a Open Source, mobile-ready Polution Detection System which uses the Planet.com API.
+
+
+### How to install
+
+##### Step 1: Clone the repository and install the dependencies
+```sh
+$ git clone https://github.com/PLA25/Website.git
+$ cd Website/Source
+$ npm install
+```
+Or download the zip file from [here](https://github.com/PLA25/Website/archive/master.zip), extract the files and open the folder Website-master/Source.
+Open your terminal on the location of the folder.
+```sh
+$ npm install
+```
+
+##### Step 2: Configure the system
+In the repository you just cloned, open the Website/Source folder and edit the config.js.example file.
+After you have filled in the information save the file as config.js in the same folder.
+Your Planet key can be found on the planet.com website, for more information about how to find your API key [click here](https://support.planet.com/hc/en-us/articles/212318178-What-is-my-API-key-)
+
+##### Step 3: Configure the database
+TO DO!
+This step will be included in an upcoming release.
+
+
+##### Step 4: Starting the system
+In the Website/Source folder execute the following command:
```

Comment:
Soort van `Zit je hier al niet in? (Zie stap 1)`.

Reply:
Beter verwoord